website/docs/5.0-upgrade-guide.html.markdown

```diff
@@ -205,7 +205,7 @@ Please follow the format in the example below for listing breaking changes in re
 
 ### `azurerm_monitor_diagnostic_setting`
 
-* The deprecated `metric.retention_policy` block has been removed.
```

Comment:
Because we're not moving `retention_policy` into `enabled_metric`. I think it'd still be helpful to know that an extra field is being removed

Reply:
It doesn't look like this comment was addressed

Reply:
I restored the line about retention policy and added the metric deprecation line

Reply:
Great! Thank you!